package.json

```diff
@@ -101,8 +101,8 @@
   },
   "scripts": {
     "test-ci": "gulp ci-tests",
-    "test": "tsc && mocha --file ./build/compiled/test/utils/test-setup.js --bail --recursive --timeout 30000  ./build/compiled/test",
```

Comment:
Was it intentional?

Reply:
Good question. Not really. It was very useful for my testing, but I did meant to ask as a part of the PR what people's thoughts on removing it are. It would be nice during test runs & CI builds to know if more than 1 test is failing, rather than having the test run stop after 1 failure.

I'll undo this if people prefer the fail early approach.

Reply:
Sometimes when test fails there are still open db connections. It makes following tests fail because they cannot open new connection. I'm not sure, but I think this is why bail option was added - to fail quickly and don't print next 200 errors which are passing when only one test is ran.

In local development I use both commands(with and without bail) so for me it doesn't really matter what is 'the standard one'.

Reply:
Interesting. I never experienced that, but I can definitely see how that would happen. I've reverted this change and will keep working without bail locally.